examples/Generic/src/main/java/org/astarteplatform/devicesdk/generic/examples/ExampleDevice.java

```diff
@@ -45,6 +38,18 @@ public static void main(String[] args) throws Exception {
     pairingUrlOpt.setRequired(true);
     options.addOption(pairingUrlOpt);
 
+    Option jwtTokenOpt =
+        new Option("t", "jwt-token", true, "The jwt token for the Astarte Register Device");
```

Comment:
@rbino do you think that `jwt-token` here is rendundant as well?

Reply:
Yep, use either `jwt` or `token`